src/components/OurValues/OurValues.css.js

```diff
@@ -15,11 +18,11 @@ const Headline1 = styled.header`
 	font-weight: bold;
 `;
 const Headline2 = styled.header`
```

Comment:
Soo I was just looking over the mark up again. Let's make the All the headings per section H2 and these sub-headings can be H3 :) 

The reason being is that users that are using accessibility devices can actually sort my an elements tag..AKA they can go to each section usually by going to an H2 :)

The <header> tag itself would more be used for a header section (Like a nav and a hero image top section). 

H2's H3's and the like are normally used for these sort of "Header texts" :)